src/main-firebase.ts

```diff
@@ -5,7 +5,7 @@ import * as express from 'express';
 import * as functions from 'firebase-functions';
 import * as admin from 'firebase-admin';
 
-admin.initializeApp();
+// admin.initializeApp();
```

Comment:
Why this?

Reply:
app was already initialised so did not need these
will remove it straight away